client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -191,10 +211,10 @@ function CheckoutSummaryFeaturesList( props ) {
 	);
 }
 
-function SupportText( { hasPlanInCart, isJetpackNotAtomic } ) {
+function SupportText( { hasPlanInCart, isJetpackNotAtomic, isPaidSite } ) {
```

Comment:
After thinking about this more, I think that basing this off whether the site has a paid plan is misleading, as it implies that the user is _purchasing_ unlimited support, which is not actually true. If they cancel their paid plan, they should get the support they paid for.

Given that subtlety, I think it might be better to use `hasEmailProductInCart` as a more explicit indication of this, and then have the alternative wording apply for that case.